backend/infrahub/graphql/mutations/computed_attribute.py

```diff
@@ -54,22 +55,17 @@ async def mutate(
         if context.branch.name == registry.default_branch:
             required_decision = PermissionDecision.ALLOW_DEFAULT
 
-        has_update_permission = False
-        for permission_backend in registry.permission_backends:
-            if has_update_permission := await permission_backend.has_permission(
-                db=context.db,
-                account_session=context.active_account_session,
-                permission=ObjectPermission(
-                    namespace=node_schema.namespace,
-                    name=node_schema.name,
-                    action=PermissionAction.UPDATE.value,
-                    decision=required_decision.value,
-                ),
-                branch=context.branch,
-            ):
-                break
-
-        if not has_update_permission:
+        permission_manager = PermissionManager(account_session=context.active_account_session)
```

Comment:
Why do we need to recreate the `PermissionManager` ? Could we access it from the context ?